ydb/docs/en/core/security/audit-log.md

```diff
@@ -1,56 +1,314 @@
 # Audit log
 
-_An audit log_ is a stream that includes data about all the operations that tried to change the {{ ydb-short-name }} objects, successfully or unsuccessfully:
+_An audit log_ is a stream of records that document the operation of the {{ ydb-short-name }} cluster. Unlike technical logs, which help detect failures and troubleshoot issues, the audit log provides data relevant to security. It serves as a source of information that answers the questions: who did what, when, and from where.
 
-* Database: Creating, editing, and deleting databases.
-* Directory: Creating and deleting.
-* Table: Creating or editing table schema, changing the number of partitions, backup and recovery, copying and renaming, and deleting tables.
-* Topic: Creating, editing, and deleting.
-* ACL: Editing.
+A single audit log record may look like this:
 
-The data of the audit log stream can be delivered to:
+```json
+{"@timestamp":"2025-11-03T18:07:39.056211Z","@log_type":"audit","operation":"ExecuteQueryRequest","database":"/my_dir/db1","status":"SUCCESS","subject":"serviceaccount@as","remote_address":"ipv6:[xxxx:xxx:xxx:xxx:x:xxxx:xxx:xxxx]"}
+```
+
+Examples of typical audit log events:
+
+* Data access through DML requests.
+* Schema or configuration management operations.
+* Changes to permissions or access-control settings.
+* Administrative user actions.
```

Comment:
Let's replace the examples with the **complete** list of what audit logs cover, along with links to details for each event type. Security teams and compliance specifications have their own requirements on what should be covered by audit logs, and it'd be helpful to let them to quickly match what we have with what they need.